### DISCUSSION TITLE
Disable for wildmenu/normal mode?

### DISCUSSION THREAD

**Discussion by victorheid** [hrsh7th/nvim-cmp#710] [Q&A]
I'm trying to disable the auto complete drop-down from the normal mode, or change the way it behaves. Can't find it in the documentation.

**Comment by Shougo** [comment]
Disabled in normal mode?
It is hard to understand.
I think nvim-cmp is enabled in both normal and command line mode.

**Reply by victorheid** [reply]
Hi!

I figure it out, by removing sources from the "command line" option, at least it did what I intended to

Thanks!
